src/pyhf/simplemodels.py

```diff
@@ -79,7 +79,7 @@ def correlated_background(
             }
         ]
     }
-    return Model(spec, batch_size=batch_size, validate=validate)
+    return Model(spec, batch_size=batch_size, validate=validate, poi_name="mu")
```

Comment:
If we're going to be changing the default behaivor of `pyhf.pdf.Model` should we just go all in and make sure that `pyhf.simplemodel` also requires users to set things? Or should we keep the `pyhf.simplemodel` experience the same?

Reply:
I think it's fair to automatically set the POI for the `simplemodels` case. There is no ambiguity there for what else the POI should be.

Reply:
We could elevate this to a user accesible argument that defaults to `mu` and pass that through here, but as you said, the POI for these models is meant to be `mu`.